documentation/documentation_config.ts

```diff
@@ -140,3 +142,11 @@ export const Snippets: Snippet[] = [
     codeFile: './snippets/dynamic_sync_table.ts',
   },
 ];
+
+export const Examples: Example[] = [
+  {
+    contentFile: './examples/column-format/column_format.md',
+    codeFiles: ['./examples/column-format/no_matchers.ts'],
```

Comment:
This is indeed what I had in mind. It just occurred to me, as we start using this, if we decide that would be useful to have content and code interspersed, we could generalize this to have a single ordered list of blocks to display, which each block would be content or code:

```
blocks: [
  {type: BlockType.Content, file: '...'],
  {type: BlockType.Code, file: '...'],
  {type: BlockType.Content, file: '...'],
  {type: BlockType.Code, file: '...'],

],
```

No need to change anything now, just something to keep in mind if we need more flexibility in what we present in the side panel.